apps/web/app/environments/[environmentId]/(actionsAndAttributes)/actions/ActionRowData.tsx

```diff
@@ -0,0 +1,34 @@
+import { timeSinceConditionally } from "@formbricks/lib/time";
+import { CodeBracketIcon, CursorArrowRaysIcon } from "@heroicons/react/24/solid";
+import { SparklesIcon } from "lucide-react";
+
+export default function ActionClassDataRow({ actionClass }) {
+  return (
+    <div className="m-2 grid h-16  grid-cols-6 content-center rounded-lg hover:bg-slate-100">
+      <div className="col-span-4 flex items-center pl-6 text-sm">
+        <div className="flex items-center">
+          <div className="h-5 w-5 flex-shrink-0 text-slate-500">
+            {actionClass.type === "code" ? (
+              <CodeBracketIcon />
+            ) : actionClass.type === "noCode" ? (
+              <CursorArrowRaysIcon />
+            ) : actionClass.type === "automatic" ? (
+              <SparklesIcon />
+            ) : null}
+          </div>
+          <div className="ml-4 text-left">
+            <div className="font-medium text-slate-900">{actionClass.name}</div>
+            <div className="text-xs text-slate-400">{actionClass.description}</div>
+          </div>
+        </div>
+      </div>
+      <div className="my-auto whitespace-nowrap text-center text-sm text-slate-500">
+        {actionClass.actionCount}
```

Comment:
the new `packages/types/v1/actionClasses.ts` in main doesn't have `actionCount` anymore to make the actionClass model more generic and also usable in public endpoints without exposing sensitive data users may not want to share. please remove the column.